src/main/scala/actorbintree/BinaryTreeSet.scala

```diff
@@ -50,51 +55,76 @@ object BinaryTreeSet {
 }
 
 
-class BinaryTreeSet extends Actor {
-  import BinaryTreeSet._
+class BinaryTreeSet extends Actor with akka.actor.ActorLogging {
+
   import BinaryTreeNode._
+  import BinaryTreeSet._
 
   def createRoot: ActorRef = context.actorOf(BinaryTreeNode.props(0, initiallyRemoved = true))
```

Comment:
We should avoid doing this. Your tree gonna have an initial removed node that is useless.

Reply:
How about a flag? 
`  var isTreeEmpty = true`
It's not nice, I know. But gets the job done

Reply:
not sure how this changes something, you still having `createRoot` that spawn a node initially removed